File: htd_info/test_server.c
#include <sys/socket.h>
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>
#include <string.h>
#include <sys/types.h>
#include <signal.h>
void htd_dpi_execute_single_request(const char* msg) {
    printf ("DEBUG:!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!Server:Got a request data:%s\n",msg);
};
char* htd_get_dpi_result() {
    return "Response Done"; 
};
int main()
{
  htd_start_server("DpiSimServer",1);

};

File: htd_info/test_server.py
#!/usr/intel/pkgs/python3/3.6.3a/modules/r1/bin/python  -B
from htd_unix_socket import *


class htd_test_server(htd_unix_socket_server):
    def __init__(self, server_name, debug_mode=0):
        htd_unix_socket_server.__init__(self, server_name)

    def request_handler(self, data):
        print(("Server:Got a request data:%s") % (data))
        self.send("PASS|Response Done")


# ---------------------------------------
serv = htd_test_server("TestServer", "SimServerSocket")
